lib/schema.js

```diff
@@ -96,28 +134,13 @@ class Metaschema {
     const errors = [];
     if (!Array.isArray(schemas)) schemas = [schemas];
 
-    schemas.sort(getProcessOrder(this.config.processOrder));
+    schemas.sort(getProcessOrder(processOrder));
     for (const schema of schemas) {
       const { type } = schema;
-      const processors = this.config.processors[type];
-
-      if (!processors) {
-        throw new TypeError(`No processors defined for type: '${type}'`);
-      }
-
-      if (processors.preprocess) {
-        for (const preprocessor of processors.preprocess) {
-          errors.push(...preprocessor(schema));
-        }
-      }
-      if (processors.validateSchema) {
-        for (const validator of processors.validateSchema) {
-          errors.push(...validator(schema));
```

Comment:
Shouldn't we validate schemas?

Reply:
That's strange, do not know why this is removed, maybe this is a result of rebase, I'll check.

Reply:
This could have happened due to the fact there is no default validator and it was previously created from "system" schemas describing `Category` and `Domain` and then injected into config.